setup.py

```diff
@@ -25,6 +25,15 @@ def read(*parts):
     },
     test_suite="runtests.runtests",
     tests_require=[
+        "django-test-plus>=1.0.11",
+        "pinax-theme-bootstrap>=7.4.0",
+        "django-appconf>=1.0.1",
+        "mock>=1.3.0",
+        "django-user-accounts>=1.3.1",
+    ],
+    install_requires=[
+        "django-appconf>=1.0.1",
+        "django-user-accounts>=1.3.1",
```

Comment:
DUA is not needed outside tests, good point. Can you tell me if a package appears in install_requires, does it also need to be included in tests_require? In other words are those requirements added together for testing or are they completely isolated?